test/e2e/e2e_suite_test.go

```diff
@@ -334,15 +334,21 @@ func init() {
 	flag.BoolVar(
 		&test.VeleroCfg.FailFast,
 		"fail-fast",
-		true,
+		test.VeleroCfg.FailFast,
 		"a switch for failing fast on meeting error.",
 	)
 	flag.BoolVar(
 		&test.VeleroCfg.HasVspherePlugin,
 		"has-vsphere-plugin",
-		false,
+		test.VeleroCfg.HasVspherePlugin,
 		"a switch for installing vSphere plugin.",
```

Comment:
Aren't these default balues? What are you trying to do here?

https://pkg.go.dev/flag#BoolVar

Reply:
It seems I can delete the default value from the Makefile.

Reply:
No, after testing, I found both of them are needed.
One option is using the `github.com/vmware-tanzu/velero/pkg/cmd/util/flag/OptionalBool`, but some of the parameters are used widely in the E2E code, need to modify many files.
Maybe we can modify it later.